charts/mirrorbits/README.md

```diff
@@ -106,6 +106,9 @@ mirrorbits add -http https://ftp.halifax.rwth-aachen.de/jenkins/ -rsync rsync://
 mirrorbits add -http https://mirror.azure.jenkins.io/ -rsync rsync://mirror-rsyncd.mirror/jenkins/ -admin-name "Jenkins Infrastructure" -admin-email "jenkinsci-infra@googlegroups.com" mirror.azure.jenkins.io
 
 mirrorbits add -http https://ftp.belnet.be/mirror/jenkins/ -rsync rsync://rsync.belnet.be -admin-name "Belnet" -admin-email "ftpmaint@belnet.be" ftp.belnet.be
+
```

Comment:
Missing triple backticks (I'm not able to do a suggestion because of escaping backticks) for closing code block

Reply:
https://github.com/jonschlinkert/remarkable/issues/146